aws/resource_aws_kinesis_firehose_delivery_stream.go

```diff
@@ -133,6 +133,122 @@ func resourceAwsKinesisFirehoseDeliveryStream() *schema.Resource {
 				},
 			},
 
+			"extended_s3_configuration": {
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
```

Comment:
Thanks, I didn't know about that attribute -- I've added it but did not have the time to add an acceptance test around it.